plugins/available/docker-machine.bash

```diff
@@ -0,0 +1,11 @@
+cite about-plugin
+about-plugin 'Helpers to get Docker setup correctly for docker-machine'
+
+# Note, this might need to be different if you use a machine other than 'dev',
+# or its configured for a different IP
+if [[ `uname -s` == "Darwin" ]]; then
+  export DOCKER_HOST="tcp://192.168.99.100:2376"
```

Comment:
I had considered that initially, but that means that the variables would only be set if `docker-machine` was started at this point.

The purpose of having the variables defined in the file like this is that they will be defined regardless of whether docker-machine or boot2docker are running. As soon as you start either of them, the variables are already set, without having to do anything.